shared/src/models/map-image.ts

```diff
@@ -15,6 +15,15 @@ export class MapImage {
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
+    /**
+     * Determines the rendering order among other mapImages.
+     * A smaller number means the mapImage is behind another one.
+     * The index can also be negative.
+     * MapImages with the same zIndex don't have a defined order.
```

Comment:
But OpenLayers seems to order them deterministically. At least I couldn't produce a state where one client displayed something different than another.